### PR TITLE
[MWPW-147480] Fix: Badge overlapping title

### PR DESCRIPTION
**Pull request by seanchoi-dev**
Resolves: [MWPW-147480](https://jira.corp.adobe.com/browse/MWPW-147480)

**Test URLs:**
- Before: https://stage--milo--adobecom.hlx.live/?martech=off
- After: https://MWPW-147480-merch-card-title-fix--milo--adobecom.hlx.live/?martech=off

For consumer's page with issue:
Before: https://main--cc--adobecom.hlx.live/pl/creativecloud
After: https://main--cc--adobecom.hlx.live/pl/creativecloud?milolibs=mwpw-147480-merch-card-title-fix

### CODE REVIEW

**Review by mirafedas — APPROVED** [review]

**Comment by yesil** [comment]
@seanchoi-dev can you add a unit test?

**Review by mirafedas — APPROVED** [review]

File: libs/blocks/merch-card/merch-card.css [hunk 1]
```diff
@@ -33,3 +33,11 @@ merch-card a[is="checkout-link"].download:not(:first-of-type),
 merch-card a[is="checkout-link"].upgrade:not(:first-of-type) {
   display: none;
 }
+
+/* Tablet and larger */
+@media screen and (min-width: 768px) {
+  .badge-merch-cards .card-heading,
+  .badge-card .card-heading {
+    margin-top: 24px;
```
Comment (overmyheadandbody) [inline libs/blocks/merch-card/merch-card.css:41]: Could you use `--spacing-s`?

**Review by mokimo — APPROVED** [review]

**Comment by Roycethan** [comment]
Issues mentioned in https://jira.corp.adobe.com/browse/MWPW-147480 are fixed @seanchoi-dev

**Review by yesil — COMMENTED** [review]
Thanks @seanchoi-dev
The fix looks good to me.
However, unfortunately we didn't have time to onboard you with tacocat.
Actually the css fix should go to https://git.corp.adobe.com/wcms/tacocat.js/blob/develop/packages/web-components/src/merch-card.css.js

And the /libs/deps/merch-card.js should be generated from develop branch once your PR is merged in tacocat.

Let's take 15-30m to go over this steps.

**Comment by seanchoi-dev** [comment]
> Thanks @seanchoi-dev The fix looks good to me. However, unfortunately we didn't have time to onboard you with tacocat. Actually the css fix should go to https://git.corp.adobe.com/wcms/tacocat.js/blob/develop/packages/web-components/src/merch-card.css.js
> 
> And the /libs/deps/merch-card.js should be generated from develop branch once your PR is merged in tacocat.
> 
> Let's take 15-30m to go over this steps.

https://git.corp.adobe.com/wcms/tacocat.js/pull/593

**Pull request merged**